File: DataAccess/Models/Team.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataAccess.Models
{
    public class Team
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string City { get; set; }

        public string Mister { get; set; }

        public string Category { get; set; }

        public string Logo { get; set; }
    }
}

File: FrontEnd/ClientApp/src/models/team.ts
export class Team {
  id: number = 0;
  name: string = "";
  city: string = "";
  mister: string = "";
  idCategory: number = 0;
  logo: string = "";

  category: string = "";

  constructor(name: string) {
    this.name = name;
  }
}
